.github/workflows/inspektor-gadget.yml

```diff
@@ -872,6 +874,20 @@ jobs:
           hash=$(sha256sum $i | cut -d' ' -f1)
           echo "${hash}  $(basename $i)" >> inspektor-gadget-${{ github.ref_name }}_checksums.txt
         done
+    - name: Install Cosign
+      uses: sigstore/cosign-installer@main
+      with:
+        # We need to cosign v2.0.0 to use keyless signing without using
+        # COSIGN_EXPERIMENTAL. This version was released recently:
+        # https://blog.sigstore.dev/cosign-2-0-released/
+        cosign-release: 'v2.0.0'
+    - name: Sign all blobs
+      shell: bash
+      run: |
+        for i in *-gadget-*-*-tar-gz/*-gadget-*-*.tar.gz inspektor-gadget-${{ github.ref_name }}.yaml inspektor-gadget-${{ github.ref_name }}_checksums.txt; do
+          base_name=$(basename $i)
+          cosign sign-blob --yes $i --output-signature="${base_name}.sig" --output-certificate="${base_name}.cert"
+        done
```

Comment:
How much time `cosign verify-blob` takes to verify binary files? Depending on that we can decide whether it makes sense to sign all of them or just the checksum file

Reply:
I check verification and it takes ~3 secs for each local-gadget binary and ~0.5 secs for the checksum file, plus the time to download the sign and cert for each one. I'm wondering if signing just the checksum file would be enough. WDYT?